web/src/lib/static/loadTasks.ts

```diff
@@ -29,7 +29,7 @@ export const loadTaskByUrlSlug = (urlSlug: string): Task => {
   return loadTaskByFileName(matchingFileName);
 };
 
-const loadTaskByFileName = (fileName: string): Task => {
+export const loadTaskByFileName = (fileName: string): Task => {
```

Comment:
why are we exporting this? I don't see it being used anywhere? i could be missing it?